opm/simulators/flow/SimulatorFullyImplicitBlackoilEbos.cpp

```diff
@@ -61,4 +65,63 @@ void outputTimestampFIP(const SimulatorTimer& timer,
     OpmLog::note(ss.str());
 }
 
+void checkSerializedCmdLine(const std::string& current,
+                            const std::string& stored)
+{
+    auto filter_strings = [](const std::vector<std::string>& input)
+    {
+        std::vector<std::string> output;
+        output.reserve(input.size());
+        std::copy_if(input.begin(), input.end(), std::back_inserter(output),
+                     [](const std::string& line)
+                     {
+                        return line.compare(0, 11, "EclDeckFile") != 0 &&
+                               line.compare(0, 8, "LoadStep") != 0 &&
+                               line.compare(0, 8, "SaveFile") != 0 &&
+                               line.compare(0, 8, "SaveStep") != 0;
+                     });
+        return output;
+    };
+
+    auto write_strings = [](const std::string& file,
+                            const std::vector<std::string>& lines)
+                         {
+                            std::ofstream of(file, std::ios::out);
+                            for (const std::string& line : lines) {
+                                of << line << '\n';
+                            }
+                         };
+    auto curr_strings = split_string(current, '\n');
+    auto stored_strings = split_string(stored, '\n');
+    std::sort(curr_strings.begin(), curr_strings.end());
+    std::sort(stored_strings.begin(), stored_strings.end());
+    curr_strings = filter_strings(curr_strings);
+    stored_strings = filter_strings(stored_strings);
+
+    const std::string curr_file = "/tmp/" + unique_path("current-%%%%");
+    const std::string stored_file = "/tmp/" + unique_path("stored-%%%%");
```

Comment:
Can we assume that `/tmp` is always available?  If not, we might want `filesystem::temp_directory_path()` instead.

Reply:
now irrelevant.